Task0.10.js

```diff
@@ -0,0 +1,42 @@
+//function findCommonCharacters(stringOne, stringTwo) {
+ // var stringOne = stringOne.split("");
+  //var stringTwo = stringTwo.split("");
+
+  //function returnSubsequence(string1,string2){
+
+    //var string1; //= stringOne.split("");
+    //var  string2; //= stringTwo.split("");
+
+    /*let resultString = ' ';
+    for (let i of string1){
+        for(let n of string2){
+            if( n === i){
+               resultString.push(n);
+            }
+        }
+    };
+    return resultString;
+  }
+         
+console.log(returnSubsequence("tanakah","mashoko"));*/
+
+
```

Comment:
Remove this please. Your final submission should be as clean as possible.

Reply:
Hi Sbonelo. Thank you for the feedback. I noticed this and i Updated the code when i resubmitted the pull request. i think you will notice my commits named " Updated Answer to ......". I realised the commented code as well as the Bad naming that i had used and changed to Task0.1.js from Task0,1,js. I am not sure if the commits can be noticed. 
Also noted is the unnecessary comments withing some of the code and will be rectified instantly.